common-util/useFetchApi.js

```diff
@@ -34,8 +34,13 @@ export const useFetchVideos = (limit = 1000) => {
       thumbnail,
       video: videoUploaded,
     } = attributes || {};
-    const imageFilename = `${process.env.NEXT_PUBLIC_API_URL}${get(thumbnail, 'data.attributes.url') || ''}`;
-    const video_url = `${process.env.NEXT_PUBLIC_API_URL}${get(videoUploaded, 'data[0].attributes.url') || ''}`;
+    const thumbnailUrl = get(thumbnail, 'data.attributes.url');
+    const imageFilename = thumbnailUrl
+      ? `${process.env.NEXT_PUBLIC_API_URL}${thumbnailUrl}`
+      : '';
+      
+    const videoUploadedUrl = get(videoUploaded, 'data[0].attributes.url');
+    const video_url = videoUploadedUrl ? `${process.env.NEXT_PUBLIC_API_URL}${videoUploadedUrl} : ''`;
```

Comment:
sry, brain not working
```suggestion
    const video_url = videoUploadedUrl ? `${process.env.NEXT_PUBLIC_API_URL}${videoUploadedUrl}` : '';
```

Reply:
lol mb I should've checked as well 😓